Description: Adds an explicit dependency on JAXB to build with Java 11
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/engine/pom.xml
+++ b/engine/pom.xml
@@ -94,6 +94,11 @@
             <artifactId>paranamer</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>debian</version>
+        </dependency>
 
         <!--
         Test dependencies
